wasm.md

```diff
@@ -125,6 +128,19 @@ Function `#unsigned` is called on integers to allow programs to use negative num
     rule <k> ( FTYPE:FValType . const VAL ) => < FTYPE > VAL        ... </k>
 ```
 
+### Text Format Conventions
+
+The text format allows the use of symbolic `identifiers` in place of `indices`.
+To resolve these `identifiers` into concrete `indices`, some grammar production are indexed by an identifier context `I` as a synthesized attribute that records the declared identifiers in each index space. We call this operation `ICov`.
+
+```k
+    syntax Int ::= #ContextLookup  ( Map , TextFormatIdx ) [function]
+                 | #asInt ( KItem ) [function]
+ // -----------------------------------------------------------------
+    rule #ContextLookup(IDS:Map, I:Int) => I
+    rule #ContextLookup(IDS:Map, ID:Identifier) => {IDS [ ID ]}:>Int 
+```
```

Comment:
We don't need `#asInt` declared anymore. Also check your formatting for `#ContextLookup`.

Reply:
removed #asInt and fixed the format